sim_pkg/simulator.py

```diff
@@ -20,78 +9,22 @@
 import time
 import re
 import numpy as np
-import csv
 from itertools import chain
-import math
 import signal
-import pandas as pd
 import functiontrace
 import logging
-# import pygame
-# file = open('time.csv','w')
-# writer = csv.writer(file)
-socket_port_pandas = pd.read_csv("port.csv", header=None)
```

Comment:
You like these temporary variables, I noticed, but note how dirty they are -- they live forever until the end of the lifetime of your process. Avoid this practice.